src/controllers/confirm.company.controller.ts

```diff
@@ -1,11 +1,24 @@
-import { Request, Response } from "express";
+import {NextFunction, Request, Response} from "express";
```

Comment:
Have you linted this pr?
I think there should be spacing before and after the curlies

Reply:
sounds like your githook is not activated, run
`find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;`